mapstruct-lombok/build.gradle

```diff
@@ -1,6 +1,5 @@
 plugins {
     id 'java'
-    id 'net.ltgt.apt' version '0.9'
```

Comment:
[Delegating IDE build/run actions to Gradle](https://www.jetbrains.com/idea/whatsnew/#v2016-3-gradle)
By read README of `net.ltgt.apt` I found this reference that resoled my problem and realized my PR is useless at all so I close it.